apps/juxtaposition-ui/src/services/juxt-web/views/ctr/root.tsx

```diff
@@ -1,17 +1,23 @@
+import type { RenderContext } from '@/services/juxt-web/views/context';
 import type { ReactNode } from 'react';
 
-function DefaultHead(): ReactNode {
+export type DefaultHeadProps = {
+	ctx: RenderContext;
+};
+
+function DefaultHead(props: DefaultHeadProps): ReactNode {
 	return (
 		<>
 			<link rel="stylesheet" type="text/css" href="/css/juxt.css" />
-			<script src="/js/debug.js"></script>
-			<script src="/js/pjax.js"></script>
-			<script src="/js/juxt.js"></script>
+			{/* Debug allows non-console browsers to have some amount of the cave API. */}
+			{props.ctx.uaIsConsole ?? true ? <></> : <script src="/js/debug.global.js"></script>}
+			<script src="/js/juxt.global.js"></script>
 		</>
 	);
```

Comment:
I had trouble understanding the logic, This should be the same outcome?
```suggestion
const addDebug = !props.ctx.uaIsConsole; // Add to non-console browsers
return (
<>
<link rel="stylesheet" type="text/css" href="/css/juxt.css" />
			{/* Debug allows non-console browsers to have some amount of the cave API. */}
			{addDebug ? <script src="/js/debug.global.js"></script> : null}
			<script src="/js/juxt.global.js"></script>
</>
);
```

Reply:
That seems more sensible yeah